packages/vm/src/cache.rs

```diff
@@ -73,7 +84,7 @@ where
         check_wasm(wasm, &self.supported_features)?;
         let checksum = save_wasm_to_disk(&self.wasm_path, wasm)?;
         let module = compile(wasm)?;
-        self.fs_cache.store(&checksum, module)?;
+        self.fs_cache.store(&checksum, &module)?;
```

Comment:
Why not store to the memory cache here as well?

Reply:
It could be done. My thinking was to not waste memory (or kick out other elements) before the first execution. But I think this can be fine tuned when we have more experience.